### DISCUSSION TITLE
thanks and request

### DISCUSSION THREAD

**Discussion by atprk** [ichi4096/vivado-on-silicon-mac#58] [General]
Thank you very much, I am a student and a Mac user and I was using the very old program ise. Thanks to you, I can also use the new version. I like to upload content to YouTube as a hobby. Can I make a video on how to install the program? I can add your link in the video so that it reaches more people.

**Comment by ichi4096** [comment]
You're welcome :-)

Sure, You're free to make any video about this You want.
